Makefile

```diff
@@ -36,3 +36,10 @@ dev:
 dircache:
 	mkdir -p directory-cache
 	wget --output-document - --quiet '$(LATEST_DIRCACHE_URL)' | tar -C directory-cache -zxf -
+
+check-targets:
+	for target in x86_64-apple-ios aarch64-apple-ios-sim aarch64-apple-ios; do \
+		if ! rustup target list | grep installed | grep $$target; then \
+			rustup target add $$target; \
+		fi \
+	done
```

Comment:
that breaks installation not using `rustup` (such as mine and all system-installed rust).

anyway, target support is actively checked by `ios/build_xcf.sh` so we can directly let it fail there